marathon-gradle-plugin/src/main/kotlin/com/malinskiy/marathon/gradle/MarathonRunTask.kt

```diff
@@ -40,10 +41,7 @@ open class MarathonRunTask @Inject constructor(objects: ObjectFactory) : Abstrac
     val flavorName: Property<String> = objects.property()
 
     @Internal
-    val applicationVariant: Property<BaseVariant> = objects.property()
-
-    @Internal
-    val testVariant: Property<TestVariant> = objects.property()
+    val applicationBundles: ListProperty<GradleAndroidTestBundle> = objects.listProperty()
```

Comment:
Instead of only running one module this, in theory, allows us to create a task to run all the modules in parallel.